### PR TITLE
Measure testing coverage on pull request

### PR DESCRIPTION
**Pull request by yhn112**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,8 +77,11 @@ jobs:
           command: pip install -e . --install-option="--buildgo"
           name: setup
       - run:
-          command: pytest ./tests
+          command: pytest --cov=hivemind ./tests
           name: tests
+      - run:
+          command: codecov
+          name: codecov
```
Comment (mryab) [inline .circleci/config.yml:84]: Better to run this for all versions just in case (codecov should be able to merge the reports)

**Review by mryab — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (yhn112) [reply .circleci/config.yml:92]: 👌

**Comment by yhn112** [comment]
I'm not sure if it's better to disable comments by codecov bot (like the one above) or not.

**Comment by justheuristic** [comment]
> I'm not sure if it's better to disable comments of codecov bot (like the one above) or not.
> The same information is available in the "Checks" tab

[weak suggestion] If you're still undecided, let's disable the comments.

**Review by mryab — APPROVED** [review]

**Pull request merged**